core/federated/RTI/rti_remote.c

```diff
@@ -344,22 +344,23 @@ static int get_num_absent_upstream_transients(federate_info_t* fed) {
 }
 
 /**
- * @brief Send MSG_TYPE_UPSTREAM_CONNECTED to the specified federate.
+ * @brief Send MSG_TYPE_UPSTREAM_CONNECTED to the specified federate, only if it is connected.
```

Comment:
There are two specified federates, so "it" is ambiguous here. Suggestion:
```
 * @brief Send MSG_TYPE_UPSTREAM_CONNECTED to the specified `destination` if it is connected to the RTI,
 * telling it that the specified `upstream` federate is also now connected.
```

Reply:
Added to the upcoming commit.